reinforcement_learning/bindings/cs/rl.net.native/rl.net.global.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+#include "rl.net.native.h"
+
+// Global exports
+extern "C" {
+    API void SayHello(void);
```

Comment:
Should we be exporting New and Delete?

Reply:
No - as we'd discussed, because we do not carry over typing information between managed and unmanaged, we would only be able to have procedures that take in void*, which will fail to call the proper destructor.

Using templates will not provide much value here, because it is unclear how to ensure the name is not mangled (it becomes impossible to P/Invoke in a compiler-portable way if there is any mangling happening), and we would still have to explicitly request instantiation for each type we project. We could make it all auto-generated on build, but it's not really worth it given that we only have so many types.